lib/helpbase.gi

```diff
@@ -498,7 +498,7 @@ InstallGlobalFunction(HELP_ADD_BOOK, function( short, long, dir )
   # (looks a bit lengthy)
   sortfun := function(a, b)
     local main, pa, pb;
-    main := ["tutorial", "reference", "changes"];
```

Comment:
Unfortunately, this PR did not apply these changes to the HPC-GAP version of the code :/. I missed that during review.

Reply:
@fingolfin you're right - I will reopen #1161 to remind me of this.

Reply:
For the record, this is now done by @fingolfin in #2054